src/modules/layerInfo/services/layerInfo.js

```diff
@@ -0,0 +1,25 @@
+/**
+* @class
+* @author costa_gi
+*/
+export class LayerInfo {
```

Comment:
Let's name the class LayerInfoResult (we want to name all of our backend response data classes "XYResult" in the future)